Assets/MixedRealityToolkit/_Core/Interfaces/BoundarySystem/IMixedRealityBoundarySystem.cs

```diff
@@ -33,28 +33,54 @@ public interface IMixedRealityBoundarySystem : IMixedRealityEventSystem, IMixedR
         /// </summary>
         bool ShowFloor { get; set; }
 
+        /// <summary>
+        /// The physics layer that the generated floor is assigned to.
+        /// </summary>
+        int FloorPhysicsLayer { get; }
```

Comment:
Per @jbienzms feedback, lets make these r/w like the other properties to allow for changing values at runtime without need to reset the system (modify and reload the profile).